mlir/include/mlir/Tools/lsp-server-support/Transport.h

```diff
@@ -147,9 +147,14 @@ class MessageHandler {
                     void (ThisT::*handler)(const Param &)) {
     notificationHandlers[method] = [method, handler,
                                     thisPtr](llvm::json::Value rawParams) {
-      llvm::Expected<Param> param = parse<Param>(rawParams, method, "request");
```

Comment:
should this has been "notification" instead of "request" all along?

Reply:
I assume it was a typo, yeah. But the string would never be printed or logged anywhere prior to this change, so it didn't matter in practice :)